winui/Avatar-view/customization.md

```diff
@@ -100,7 +100,7 @@ avatarView.Background = new SolidColorBrush(Colors.Bisque);
 
 ## Gradient background
 
-Use the `LinearGradientBrush` or `RadialGradientBrush` for the Background property to specify a range of colors to apply the gradient background for the AvatarView as shown in the following code example.
+Use the [LinearGradientBrush](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.media.lineargradientbrush?view=winrt-22621) or [RadialGradientBrush](https://learn.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.media.radialgradientbrush?view=winui-2.8) for the [Background](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.control.background?view=winrt-22621) property to specify a range of colors to apply the gradient background for the AvatarView as shown in the following code example.
```

Comment:
@MadhumithanJayapathi No need to add link for gradient brush

Reply:
@Manivannan-E Removed the links for gradient brush.